macs2.wdl

```diff
@@ -30,8 +30,15 @@ task PeakCalling {
         String sampleName
         String format = "AUTO"
         Boolean nomodel = false
+        String gensz = "hs"
+        Int extsize
```

Comment:
Please make this optional.

Reply:
You are right, I ll make all of them optional